src/tensor_ops.jl

```diff
@@ -1,3 +1,15 @@
+# Deprecate `*` and `'*` for tensor multiplication
```

Comment:
Remove the errors you mean?

I thought there would be some fancy way to stop functions from being able to operate on specific types...